maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/IncrementalifyMojo.java

```diff
@@ -70,18 +76,26 @@ public class IncrementalifyMojo extends AbstractVersionsUpdaterMojo {
     @Component
     private BuildPluginManager pluginManager;
 
+    @Inject public IncrementalifyMojo(RepositorySystem repositorySystem, org.eclipse.aether.RepositorySystem aetherRepositorySystem, Map<String, Wagon> wagonMap, Map<String, ChangeRecorder> changeRecorders) {
+        super(repositorySystem, aetherRepositorySystem, wagonMap, changeRecorders);
+    }
+
     @Override public void execute() throws MojoExecutionException, MojoFailureException {
         File dotMvn = new File(project.getBasedir(), ".mvn");
         File extensionsXml = new File(dotMvn, "extensions.xml");
         if (extensionsXml.isFile()) {
             throw new MojoFailureException("Editing an existing " + extensionsXml + " is not yet supported");
         }
-        VersionRange any;
         ArtifactVersions gclmeVersions;
+        try {
+            gclmeVersions = getHelper().lookupArtifactVersions(getHelper().createDependencyArtifact("io.jenkins.tools.incrementals", "git-changelist-maven-extension", "[0,)", "type", null, null), true);
+        } catch (VersionRetrievalException x) {
+            throw new MojoExecutionException(x.getMessage(), x);
+        }
+        VersionRange any;
         try {
             any = VersionRange.createFromVersionSpec("[0,)");
-            gclmeVersions = getHelper().lookupArtifactVersions(getHelper().createDependencyArtifact("io.jenkins.tools.incrementals", "git-changelist-maven-extension", any, "type", null, null), true);
-        } catch (ArtifactMetadataRetrievalException | InvalidVersionSpecificationException x) {
+        } catch (InvalidVersionSpecificationException x) {
```

Comment:
Adapting to random compilation failures. Stepped through the new code in the debugger to make sure it was working.